tests/loader_regression_tests.cpp

```diff
@@ -3569,3 +3569,38 @@ TEST(DuplicateRegistryEntries, Drivers) {
         "\" from registry \"HKEY_LOCAL_MACHINE\\SOFTWARE\\Khronos\\Vulkan\\Drivers\" to the list due to duplication"));
 }
 #endif
+
+#if defined(__linux__) || defined(__FreeBSD__) || defined(__OpenBSD__)
+// Check that valid symlinks do not cause the loader to crash
```

Comment:
Aside from not crashing, valid symlinks are supposed to actually work right?

Reply:
Yes, since a symlink should just 'resolve' to a manifest json file, there is no technical reason why they can't be used (and the test shows that they do work.)

Reply:
I just redid the two tests here by splitting them into 4, where the matrix is valid & invalid symlink by 'found in regular search paths' and 'symlink is directly in XDG_CONFIG_DIRS